backoffice/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:20-alpine AS build
+FROM node:24-alpine AS build
```

Comment:
@andiempett I think this one will need changes in other places too, but it's promising that the checks passed on this.

Reply:
@WillGibson 
yeah it does
but i would personally go with 22 as thats the LTS one, unless there is some pressing security vulnerability